packages/core/src/components/MessageItem/index.tsx

```diff
@@ -122,6 +126,12 @@ export class MessageItem extends React.Component<Props & WithStylesProps> {
       warning && styles.container_warning,
     );
 
+    const formatedTitle = disableTitleTranslation ? (
```

Comment:
@stefhatcher Any reason `title` doesn't support nodes? Seems a bit odd to hardcode this.

Reply:
it's used in other places that are string only

Reply:
👍